lib/awful/layout/suit/tile.lua

```diff
@@ -317,6 +324,9 @@ end
 function tile.left.mouse_resize_handler(c, corner, x, y)
     return mouse_resize_handler(c, corner, x, y, "left")
 end
+function tile.left.skip_gap(nclients, t)
+    return tile.skip_gap(nclients, t)
+end
```

Comment:
`tile.left.skip_gap = tile.skip_gap` seems simpler to me (same for similar places here)